en/restaurant/create-restaurant.md

```diff
@@ -0,0 +1,128 @@
+
+- [General](#General)
+- [Options](#Options)
+- [Fulfillment](#Fulfillment)
+- [Settlement](#Settlement)
+
+### General
+
+On this `General` tab, you can modify general informations about your restaurant :
+
+- Displayed to clients :
+  - Name of the Restaurant on CoopCycle
+  - Type of Business
+  - Adress
+  - Description of your restaurant 
+  - Webd
+  
+- Not displayed to clients :
+  - Website
+  - Telephone number
+  - SIRET number
+  - VAT number
+  - RCS number
+  
+### Options
+
+4 options have to be decided on this tab :
+
+- Restaurant is exclusive which means that it is only available on CoopCycle
+- Restaurant is featured on homepage 
+- Restaurant also accepts quotes (e.g.:catering)
+- Enable Deposit-Refund system
+
+### Fulfillment
+
+#### Schedules
+On this page, you can chose between two types of activity, or chose both :
+- Collect, which means that the customer come to the restaurant to take-away his or her order
+- Delivery, which means that the cooperative delivers the order to the customer
+
+For each activity, you can edit a schedule which determines when customers can receive their orders or come take them at the restaurant. 
+
+For each schedule you have to chose between *As soon as possible* and *time slot* :
+- *As soon as possible* : The customer receives his or her order as soon as possible. On his or her application, the customer during his or her order sees : "in XX minutes"
+- *Time slot* : The customer choses when he or her receives the order. On his or her application, the customer during his or her order sees : "Between XX PM/AM and XX PM/AM"
+
+For each schedule, you have to chose with the administrator if the restaurant-owner has the right to change these schedules. Click on `allow editing` if it is the case.
+
+#### Global options
+
+- Define a delivery perimeter
+- Define, if needed, an additional delay before ordering. For instance, if you put 1 in the `day` field, customers have to order the day before.
+- Define, if needed, a number of shipping choices
+
+### Settlement
+
+You will find on this page **the terms of the contract** between the local delivery collective and the restaurant. They cover the following aspects :
+- Delivery price :
+    - Part to be paid by the restaurant ("Amount charged by the platform" for a fixed cost and "Platform fee for deliveries" for a proportionnal fee). It is also possible to create your own rule of pricing and apply it in this page. Please see the documentation about pricing in administrator's guide/logistics.
+    - Part to be paid by the final client.("Amount paid by the customer" for a fixed cost and "Fees paid by the customer" for a proportionnal fee). It is also possible to create your own rule of pricing and apply it in this page. 
+- Minimum basket amount
+- Platform fees for take-away.
+- Payment fees (Stripe): paid by the platform or the restaurant owner
```

Comment:
Remove this line, this is deprecated: restaurant will _always_ pay the fees

Reply:
Ok ( I Imagine that you speak about Stripe fees)